README.md

```diff
@@ -15,37 +15,46 @@ $ npm install --save shopify-api-node
 
 ## Usage:
 
-This module exports a constructor function which takes three or two arguments
-depending if you want to use it for private or public apps.
+This module exports a constructor function which takes an object with properties, requirements depending if you want to use it for private or public apps.
```

Comment:
Keep the 80 chars per line limit and use "options object" instead of "object with properties".
You can cut the sentence there at "options object". I would like to see the list of allowed options but we add them later.